modules/vault-iam/main.tf

```diff
@@ -0,0 +1,44 @@
+resource "aws_iam_user" "vault_user" {
+  name = "vault_user"
+  tags = {
+    "user" = "vault"
+  }
+}
+
+resource "aws_iam_access_key" "vaultkey" {
+  user = "${aws_iam_user.vault_user.name}"
+}
```

Comment:
I'm generally against using Terraform to pass around these credentials. I could be convinced to use `count` and make this optional.

Reply:
Fair enough. I will make this as optional.

Reply:
I make this optional now with the default behaviour not creating any keys.